src/data/lang/__init__.py

```
@@ -0,0 +1,236 @@
+
+def _parse_lang_value(v: Any) -> Any:
+    """统一解析语言节点（核心逻辑抽取）"""
+    if not isinstance(v, dict):
+        return v
+
+    if v.get("string") is not None:
+        return v.get("string")
+    if v.get("pattern") is not None:
+        try:
+            return re.compile(v.get("pattern"))
+        except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)